DfE.FindInformationAcademiesTrusts/Services/Export/Builders/AcademiesBuilder.cs

```diff
@@ -0,0 +1,131 @@
+                ExportHelpers.SchoolName,
+                ExportHelpers.Urn,
+                ExportHelpers.LocalAuthority,
+                "Type",
+                "Rural or Urban",
+                ExportHelpers.DateJoined,
+                "Current Ofsted Rating",
+                ExportHelpers.BeforeOrAfterJoiningHeader,
+                "Date of Current Ofsted",
+                "Previous Ofsted Rating",
+                ExportHelpers.BeforeOrAfterJoiningHeader,
+                "Date of Previous Ofsted",
+                "Phase of Education",
+                ExportHelpers.AgeRange,
+                "Pupil Numbers",
+                "Capacity",
+                "% Full",
+                "Pupils eligible for Free School Meals",
+                "LA average pupils eligible for Free School Meals",
+                "National average pupils eligible for Free School Meals",
+                "Pupils eligible for Free School Meals"
+            };
+
+            WriteHeaders(headers);
+
+            return this;
+        }
+
+        public AcademiesBuilder WriteRows(AcademyDetails[] academies, AcademyOfsted[] academiesOfstedRatings,
+            AcademyPupilNumbers[] academiesPupilNumbers,
+            AcademyFreeSchoolMealsServiceModel[] academiesFreeSchoolMeals)
+        {
+            foreach (var details in academies)
+            {
+                var ofstedData = academiesOfstedRatings.Single(x => x.Urn == details.Urn);
+                var pupilData = academiesPupilNumbers.Single(x => x.Urn == details.Urn);
+                var freeMealsData = academiesFreeSchoolMeals.Single(x => x.Urn == details.Urn);
+
+                GenerateAcademyRow(details, ofstedData, pupilData, freeMealsData);
+                AddRow();
+            }
+
+            return this;
+        }
+
+        private void GenerateAcademyRow(
+            AcademyDetails academy,
+            AcademyOfsted ofstedData,
+            AcademyPupilNumbers pupilNumbersData,
+            AcademyFreeSchoolMealsServiceModel freeSchoolMealsData)
+        {
+            var previousRating = ofstedData.PreviousOfstedRating;
+            var currentRating = ofstedData.CurrentOfstedRating;
+            var percentageFull = ExportHelpers.CalculatePercentageFull(pupilNumbersData.NumberOfPupils, pupilNumbersData.SchoolCapacity);
+
+            SetTextCell(CurrentRow, 1, academy.EstablishmentName ?? string.Empty);
+            SetTextCell(CurrentRow, 2, academy.Urn);
+            SetTextCell(CurrentRow, 3, academy.LocalAuthority ?? string.Empty);
+            SetTextCell(CurrentRow, 4, academy.TypeOfEstablishment ?? string.Empty);
+            SetTextCell(CurrentRow, 5, academy.UrbanRural ?? string.Empty);
+
+            SetDateCell(CurrentRow, 6, ofstedData.DateAcademyJoinedTrust);
+
+            SetTextCell(CurrentRow, 7, currentRating.OverallEffectiveness.ToDisplayString(true));
+            SetTextCell(CurrentRow, 8,
+                ExportHelpers.IsOfstedRatingBeforeOrAfterJoining(
```

Comment:
If we used the service model object we would have access to `WhenDidCurrentInspectionHappen` which gives a `BeforeOrAfterJoining` enum. This enum is currently translated to a display string in `DfE.FindInformationAcademiesTrusts.Pages.Trusts.Ofsted.SingleHeadlineGradeCellsModel.TagLabel` but we could move that to a new `BeforeOrAfterJoiningExtensions` class with a `ToDisplayString` method so it can be shared between the two - that way they won't deviate like they already have (the casing used to be the same but is now different)